src/main/resources/db.migrations/v1__create_employee_table.sql

```diff
@@ -0,0 +1,16 @@
+
+CREATE TABLE employee_model (
+                                id BIGINT AUTO_INCREMENT PRIMARY KEY,
+                                name VARCHAR(255) NOT NULL,
+                                gender VARCHAR(10),
```

Comment:
add validation on the values like the regex you have in the request but foe the db